src/PerfView/OtherSources/XMLStackSource.cs

```diff
@@ -113,6 +113,85 @@ public static void WriteStacks(StackSource source, XmlWriter writer)
         }
     }
 
+    public class JsonStackSourceWriter
+    {
+        public static void WriteStackViewAsZippedJson(StackSource source, string fileName, Action<TextWriter> additionalData = null)
+        {
+            if (File.Exists(fileName))
+                File.Delete(fileName);
```

Comment:
curly braces missing, not sure if there's a standard in this repo yet, but it'd be great to have a build failure for this one.